kettle-plugins/kafka/src/main/java/org/pentaho/big/data/kettle/plugins/kafka/KafkaStepAnalyzer.java

```diff
@@ -0,0 +1,52 @@
+/*! ******************************************************************************
+ *
+ * Pentaho Data Integration
+ *
+ * Copyright (C) 2002-2019 by Hitachi Vantara : http://www.pentaho.com
+ *
+ *******************************************************************************
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ ******************************************************************************/
+package org.pentaho.big.data.kettle.plugins.kafka;
+
+import org.pentaho.di.i18n.BaseMessages;
+import org.pentaho.dictionary.DictionaryHelper;
+
+import static org.pentaho.dictionary.DictionaryConst.CATEGORY_DATASOURCE;
+import static org.pentaho.dictionary.DictionaryConst.CATEGORY_MESSAGE_QUEUE;
+import static org.pentaho.dictionary.DictionaryConst.LINK_CONTAINS_CONCEPT;
+import static org.pentaho.dictionary.DictionaryConst.LINK_PARENT_CONCEPT;
+import static org.pentaho.dictionary.DictionaryConst.NODE_TYPE_EXTERNAL_CONNECTION;
+
+class KafkaStepAnalyzer {
+  private KafkaStepAnalyzer() { }
+
+  private static final Class<KafkaStepAnalyzer> PKG = KafkaStepAnalyzer.class;
+
+  static final String NODE_TYPE_KAFKA_TOPIC = BaseMessages.getString( PKG, "KafkaStepAnalyzer.KafkaTopic" );
+  static final String NODE_TYPE_KAFKA_SERVER = BaseMessages.getString( PKG, "KafkaStepAnalyzer.KafkaServer" );
```

Comment:
I'm not sure if the node type names should be localized.  If the graph is programmatically queried or manipulated, the types would need to be fixed identifiers.  Changing based on locale could break that.

Reply:
good point.  the others aren't localized either.  I'll change it